python/agent.py

```diff
@@ -111,14 +163,19 @@ def download_agent_path(
 
 
 def download_sandbox_files(
-    agent_id: str, sandbox: List[dict], output_base_path: str, patterns_to_download: List[str] = []
-) -> List[dict]:
+    agent_id: str, sandbox: List[dict], output_base_path: str, patterns_to_download: List[str]
+) -> None:
+    if not sandbox:
+        logger.warning("download_sandbox_files: input file list is empty!")
+        return
     if not os.path.exists(output_base_path):
         os.makedirs(output_base_path)
 
     for task_file in sandbox:
+        # print("download_sandbox_files: file_path = " + task_file["path"])
```

Comment:
Let's replace these comments with `log.info()`.

Reply:
I have added these print() only for local debugging purposes. It is absolutely not needed in Prod logging because of huge output but it is very useful to troubleshoot log files downloading issues when developing new bundles which use log downloading. Because this low level function is shared across all bundles.